bukkit/src/main/java/dev/aurelium/auraskills/bukkit/hooks/BukkitLuckPermsHook.java

```diff
@@ -32,17 +32,26 @@ public class BukkitLuckPermsHook extends LuckPermsHook implements Listener {
 
     private final String prefix = "auraskills.multiplier.";
     private final Map<UUID, Set<String>> permissionCache = new ConcurrentHashMap<>();
+    private final boolean optimizePermissionLookup;
 
     public BukkitLuckPermsHook(AuraSkillsPlugin plugin, ConfigurationNode config) {
         super(plugin, config);
 
+        this.optimizePermissionLookup = getConfig().node("optimize_permission_lookup").getBoolean();
```

Comment:
Use the constructor param config instead of getConfig(), also pass true as default value for getBoolean.